From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Sat, 2 Oct 2021 12:03:48 +0200
Subject: [PATCH 02/10] drm: Increase DRM_OBJECT_MAX_PROPERTY by 18.

The warning poped up, it says it increase it by the number of occurence.
I saw it 18 times so here it is.
It started to up since commit
   2f425cf5242a0 ("drm: Fix oops in damage self-tests by mocking damage property")

Increase DRM_OBJECT_MAX_PROPERTY by 18.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Link: https://lkml.kernel.org/r/20211005065151.828922-1-bigeasy@linutronix.de
---
 include/drm/drm_mode_object.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/drm/drm_mode_object.h
+++ b/include/drm/drm_mode_object.h
@@ -60,7 +60,7 @@ struct drm_mode_object {
 	void (*free_cb)(struct kref *kref);
 };
 
-#define DRM_OBJECT_MAX_PROPERTY 24
+#define DRM_OBJECT_MAX_PROPERTY 42
 /**
  * struct drm_object_properties - property tracking for &drm_mode_object
  */
